### ISSUE TITLE
Mulighet for å registrere ekstra epostadresser

### ISSUE THREAD

**Issue by asgeirn** [FINTLabs/fint-kunde-portal-backend#32]
Ettersom vi krever Id-porten for registrering i kundeportalen er det flere personer som burde fått epost som ikke kan registreres.

Det bør derfor være mulig å registrere ekstra varslingsmottakere for en organisasjon, for eksempel knyttet til adapter eller klient.

**Comment by fsjovatsen** [comment]
@asgeirn Ettersom vi krever Id-porten for registrering i kundeportalen er det flere personer som burde fått epost som ikke kan registreres???????

**Comment by asgeirn** [comment]
Ja, personer som ikke har norsk fødselsnummer og dermed ikke kan registrere seg i kundeportalen - men som likevel burde kunne varsles.
